pages/common/csvclean.md

```diff
@@ -0,0 +1,12 @@
+# csvclean
+
+> Finds and cleans common syntax errors in CSV files.
+> Included in csvkit.
```

Comment:
Hmm, I'm not sure. I'm tempted to stick with the less wordy options because, by Markdown standards, these two separate lines actually combine into one long line.